src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -642,8 +642,7 @@ public static String sanitizeName(String type, String name) {
 			return null;
 		}
 
-		// only look for ":" and "/" as of now
-		Pattern p = Pattern.compile("[\\\\:\\/\\>\\<]");
+		Pattern p = Pattern.compile("[\\\\:\\/\\>\\<\\*\\|\\?]");
```

Comment:
Please explain what this should be matching?  Sometimes a regex is self explanatory but this could use some description.